File: java/logic-1/love6.java
/* The number 6 is a truly great number. Given two int values, a and b, 
 * return true if either one is 6. Or if their sum or difference is 6. Note: 
 * the function Math.abs(num) computes the absolute value of a number.
 */
public boolean love6(int a, int b) {
    if(a == 6 || b == 6 || a + b == 6 || Math.abs(a - b) == 6)
        return true;
          
    return false;
}

File: python/logic-1/love6.py
# The number 6 is a truly great number. Given two int values, a and b, return 
# True if either one is 6. Or if their sum or difference is 6. Note: the 
# function abs(num) computes the absolute value of a number.
def love6(a, b):
    if a == 6 or b == 6:
        return True
          
    return a + b == 6 or abs(a - b) == 6
